### PR TITLE
Brigmedic hardsuit update

### PR DESCRIPTION
**Pull request by JustAdler**
## Short description

Updating the brigmedic hardsuit to be in line with the rest of security hardsuits, with added caustic protection.

https://discord.com/channels/1272545509562777621/1316102516877820004

## Why we need to add this

Brigmedic often ditches his suit for the security one.

## Media (Video/Screenshots)

image

## Checks

- [ ] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: aedler
- tweak: Updated the brigmedic suit values to be more in line with the security hardsuits.

### CODE REVIEW

**Comment by JustAdler** [comment]
who the hell is YAML Linter and what did he do to my check

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml [hunk 1]
```diff
@@ -256,15 +256,18 @@
...
   - type: Armor
     modifiers:
       coefficients:
-        Blunt: 0.8
-        Slash: 0.8
-        Piercing: 0.7
+        Blunt: 0.6
+        Slash: 0.6
+        Piercing: 0.6
+        Caustic: 0.5
```
Comment (Rinary1) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml]: Why more protection than a security suit? The whole point of a brig medic suit is that it shouldn't slow you down, but it shouldn't have the same protection as a security suit.

**Review by Rinary1 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml (hunk 1 above)
Comment (JustAdler) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml]: Farewell, i could make it light weight with less armor. Besides, it was weaker and considerably slower than the hard suit prior

Comment (fqqf) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml]: i agree, this actually makes a lot more sense, just make brigmedic be like standard sec suit and make it maybe 5% faster

**Pull request merged**